client/src/components/SupportTools/Audit.tsx

```diff
@@ -30,52 +30,89 @@ const Audit = ({ electionId }: { electionId: string }) => {
   } = election.data
 
   return (
-    <div style={{ width: '100%' }}>
-      <Breadcrumbs>
-        <Link to={`/support/orgs/${organization.id}`}>{organization.name}</Link>
-      </Breadcrumbs>
-      <H2>{auditName}</H2>
-      <Column>
+    <div
+      style={{
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '20px',
+        marginTop: '20px',
+      }}
+    >
+      <Row>
         <div
           style={{
-            alignItems: 'center',
             display: 'flex',
-            marginBottom: '10px',
+            flexDirection: 'column',
+            gap: '20px',
+            alignItems: 'flex-start',
+            width: '100%',
           }}
         >
-          <Tag large style={{ marginRight: '10px' }}>
-            {prettyAuditType(auditType)}
-          </Tag>
-          <AnchorButton
-            href={`/api/support/elections/${id}/login`}
-            icon="log-in"
-            intent="primary"
+          <div
+            style={{
+              display: 'flex',
+              width: '100%',
+              alignItems: 'flex-start',
+              justifyContent: 'space-between',
+            }}
           >
-            Log in as audit admin
-          </AnchorButton>
+            <H2 style={{ margin: '0', maxWidth: '80%' }}>{auditName}</H2>
+            <AnchorButton
+              href={`/api/support/elections/${id}/login`}
+              icon="log-in"
+              intent="primary"
+            >
+              Log in as audit admin
+            </AnchorButton>
+          </div>
         </div>
-        <div style={{ marginBottom: '10px' }}>
-          <RoundsTable electionId={electionId} rounds={rounds} />
+      </Row>
+      <Row>
+        <div style={{ display: 'flex', gap: '6px', alignItems: 'center' }}>
+          <Tag large>{prettyAuditType(auditType)}</Tag> for
+          <Breadcrumbs>
+            <Link to={`/support/orgs/${organization.id}`}>
+              {organization.name}
+            </Link>
+          </Breadcrumbs>
```

Comment:
The reason I didn't in his case is that it doesn't really look like breadcumbs, since it's just a single link to the organization. I felt this put the relevant descriptive info in one place. But, I don't feel too strongly about it, so I dont mind reverting

Reply:
![Screenshot 2024-12-16 at 3 07 58 PM](https://github.com/user-attachments/assets/22ddd55b-3986-40db-a6a9-8769628f512a)

[Updated](https://github.com/votingworks/arlo/pull/2094/commits/5298034c2bfb42efda3abc4025a2b2deac9f58f8)